### PR TITLE
Quickfix for MuleLog issues with multiple accounts "all" selection

### PR DESCRIPTION
**Pull request by SonnyDAscola**
Quickfix for MuleLog issues with multiple accounts "all" selection - See 
https://github.com/kolton/d2bot-with-kolbot/issues/1835 
https://github.com/kolton/d2bot-with-kolbot/issues/1834

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,31 @@
+# D2BS with HotFixes and extended functionality
```
Comment (jaenster) [inline README.md]: I think you dont want to add this text in the original, as if its in the original it isnt a fork anymore

File: d2bs/kolbot/D2BotFollow.dbj [hunk 2]
```diff
@@ -12,7 +12,7 @@ var StarterConfig = {
 	CDKeyInUseDelay: 5, // Minutes to wait before connecting again if CD-Key is in use. SwitchKeys overrides this!
 	ConnectingTimeout: 20, // Seconds to wait before cancelling the 'Connecting...' screen
 	PleaseWaitTimeout: 30, // Seconds to wait before cancelling the 'Please Wait...' screen
-	WaitInLineTimeout: 60, // Seconds to wait before cancelling the 'Waiting in Line...' screen
+	WaitInLineTimeout: 80, // Seconds to wait before cancelling the 'Waiting in Line...' screen
```
Comment (jaenster) [inline d2bs/kolbot/D2BotFollow.dbj]: I dont see a reason for this entire file to be edited for this specific pull request

File: d2bs/kolbot/libs/bots/Follower.js [hunk 3]
```diff
@@ -886,6 +886,9 @@ WPLoop:
 
 		delay(100);
 	}
+	// Following two lines added by Sonny
```
Comment (jaenster) [inline d2bs/kolbot/libs/bots/Follower.js]: What has this to do with mule logging?

File: d2bs/kolbot/libs/common/Config.js [hunk 4]
```diff
@@ -347,6 +347,9 @@ var Config = {
 	Rakanishu: {
 		KillGriswold: false
 	},
+	Wakka: {
```
Comment (jaenster) [inline d2bs/kolbot/libs/common/Config.js]: What has this to do with mule logging?

File: d2bs/kolbot/libs/common/Config.js [hunk 5]
```diff
@@ -428,7 +431,9 @@ var Config = {
 		GameLength: 3
 	},
 	Follower: {
-		Leader: ""
+		Leader: "",
```
Comment (jaenster) [inline d2bs/kolbot/libs/common/Config.js]: What has this to do with mule logging?

File: d2bs/kolbot/libs/config/Amazon.js [hunk 6]
```diff
@@ -152,6 +152,7 @@ function LoadConfig() {
 		Config.BaalHelper.KillBaal = true; // Kill Baal. If set to false, you must configure Config.QuitList or the bot will wait indefinitely.
 		Config.BaalHelper.SkipTP = false; // Don't wait for a TP, go to WSK3 and wait for someone to go to throne. Anti PK measure.
 	Scripts.Follower = false; // Script that follows a manually played leader around like a merc. For a list of commands, see Follower.js
+		Config.Follower.ExitTimeout = 5;
```
Comment (jaenster) [inline d2bs/kolbot/libs/config/Amazon.js]: What has this to do with mule logging?

File: d2bs/kolbot/libs/config/Assassin.js [hunk 7]
```diff
@@ -152,6 +152,7 @@ function LoadConfig() {
 		Config.BaalHelper.KillBaal = true; // Kill Baal. If set to false, you must configure Config.QuitList or the bot will wait indefinitely.
 		Config.BaalHelper.SkipTP = false; // Don't wait for a TP, go to WSK3 and wait for someone to go to throne. Anti PK measure.
 	Scripts.Follower = false; // Script that follows a manually played leader around like a merc. For a list of commands, see Follower.js
+		Config.Follower.ExitTimeout = 5;
```
Comment (jaenster) [inline d2bs/kolbot/libs/config/Assassin.js]: What has this to do with mule logging?

File: d2bs/kolbot/libs/config/Sorceress.js [hunk 8]
```diff
@@ -152,7 +152,7 @@ function LoadConfig() {
 		Config.BaalHelper.KillBaal = true; // Kill Baal. If set to false, you must configure Config.QuitList or the bot will wait indefinitely.
 		Config.BaalHelper.SkipTP = false; // Don't wait for a TP, go to WSK3 and wait for someone to go to throne. Anti PK measure.
 	Scripts.Follower = false; // Script that follows a manually played leader around like a merc. For a list of commands, see Follower.js
-
+		Config.Follower.ExitTimeout = 5;
```
Comment (jaenster) [inline d2bs/kolbot/libs/config/Sorceress.js]: What has this to do with mule logging? Removed here an enter (by mistake) too

File: d2bs/kolbot/libs/config/Paladin.js [hunk 9]
```diff
@@ -152,6 +152,7 @@ function LoadConfig() {
 		Config.BaalHelper.KillBaal = true; // Kill Baal. If set to false, you must configure Config.QuitList or the bot will wait indefinitely.
 		Config.BaalHelper.SkipTP = false; // Don't wait for a TP, go to WSK3 and wait for someone to go to throne. Anti PK measure.
 	Scripts.Follower = false; // Script that follows a manually played leader around like a merc. For a list of commands, see Follower.js
+		Config.Follower.ExitTimeout = 5;
```
Comment (jaenster) [inline d2bs/kolbot/libs/config/Paladin.js]: What has this to do with mule logging?

**Comment by SonnyDAscola** [comment]
First of all: thanks for the review, I'll change some of these lines back as well as making the code clean again!

I have to apologize for the trouble of changes without context, I rebased the Branch to a later commit on Master, where i implemented a feature called “ExitTimeout“ to prevent RD on multiple followers.

Will branch these changes and rebase the mulelogger branch back to earlier state to clean up not-topic code.

README was modified to not look like “the real deal“ - would have set it to private if I would have been able to while still sharing it with you guys.

Will do a re-review and get back here! Once my net wants to work properly again.

Thank you very much!

**Comment by SonnyDAscola** [comment]
Closing due to different git setup, will provide new pull request soon

**Pull request closed without merging**